index.bs

```diff
@@ -2916,6 +2916,214 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method {#sctn-signal}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signal(PublicKeyCredentialSignalOptions options);
```

Comment:
Does "`signal()`" leave the door open for a wider range of functionality? I kinda liked "`report()`" (maybe "`sendReport()`" instead though tbh 🤔) as [initially proposed via the explainer](https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Report-API-explainer). But I wonder if the current proposal was born out of subsequent conversations around naming how a "signal" might be more flexible than "report".

Reply:
Our devrel team asked it to be renamed to avoid confusion with the [Reporting API](https://w3c.github.io/reporting/).

Reply:
One other name suggestion would be "cleanup"

Reply:
I'd leave it more generic (I preferred "report" as well but "signal" seems fine) since e.g. just a metadata update signal is not actually cleanup.

I also think we may want non-cleanup signals in the future, such as "I the RP just accepted this credential" - which providers may find useful to simplify UI with MRU logic, for example.

Reply:
`signal()` it is then. On to bikeshedding other things, marking as resolved.